docs/terminal/amberleyoakey.md

```diff
@@ -11,89 +11,96 @@
 
 | Departure | Initial Constraint | Route |
 | --------- | ----------| --------- |
-| Northern 1 | `F190` at BINUP | BINUP MURJO BOBED LESKO MOSSI/ADNUK (Transition) |
-| Central 1 | `F190` at BINUP | BINUP MURJO BOBED VIRGE |
-| Byron 7 | `F190` at COWIE | COWIE KIWEE ZANET LOTMA SEMAJ |
+| Northern 3 | `F190` at BINUP | BINUP MURJO BOBED LESKO MOSSI/ADNUK (Transition) |
+| Northern 4 | `F200` at ANKED | ANKED JEZZA BOBED LESKO MOSSI/ADNUK (Transition) |
+| Central 3 | `F190` at BINUP | BINUP MURJO BOBED VIRGE |
+| Central 4 | `F200` at ANKED | ANKED JEZZA BOBED VIRGE |
+| Byron 1 | `F190` at COWIE | COWIE KIWEE ZANET LOTMA SEMAJ |
+| Byron 2 | `F200` at TUGUN | TUGUN LAGOB ZANET LOTMA SEMAJ |
 
-!!! phraseology
-    WOLF03 was assiged the BYRON7 departure with their intial clearance.  
-    **AMB ADC**: "WOLF03, make left turn, reach `F190` by COWIE, cleared for takeoff"  
-    **WOLF03**: "Make left turn, reach `F190` by COWIE, cleared for takeoff, WOLF03"
+The Coded Departures are not present in vatSys, however ADC will write the cleared Coded Departure in the Global Ops Field and include the relevant tracking points in the aircraft's route.
 
-!!! note
-    AMB ACD will record the name of the coded departure in the Global Ops Field of the FDR and amend the flight plan route with all necessary tracking points. 
+!!! abstract "Reference"
+    The Coded Departures are detailed in: [YAMB FIHA AD2 Supp Section 6.2.25](https://ais-af.airforce.gov.au/australian-aip){target=new}
+
+## Civil Training Area
+Civil operators (including Australian Air Force Cadets) conduct flying operations within the South Western Training Area (SWTA), located between the 10DME and 20DME arcs to the southwest of the AMB CTR. The SWTA is divided in half by the Rosewood-Mt Walker-Aratula Road into SWTA Alpha (north) and SWTA Bravo (south). The training area is classified Class G airspace but some pilots may request to climb into CTA to facilitate flight training (such as stall practice).
 
 !!! tip
-    Charts for the Coded Departures may be in YAMB AD2 Supplements 6.2.25.11 available here: [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
+    Consider restricting an aircraft operating in the CTA above the SWTA to either SWTA Alpha or Bravo where doing so would provide lateral segregation from other traffic operating in the AMB CTA.
 
-## Charts
-!!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
+### Outbound Aircraft
+Aircraft outbound from YAMB for the SWTA will exit the CTR at `A025` via the following waypoints:
+
+| **Duty Runway** | **Routing** |
+|-----------------|-----------|
+| 15              | MTWK      |
+| 33              | CLVT      |
+| 04 or 22        | As required |
+
+Once the aircraft is clear of the zone, cancel their control service.
```

Comment:
Is this not implied?

Reply:
> Is this not implied?

It should be, I was just struggling to find something to write. I'll remove